### DISCUSSION TITLE
What should I install for my React v18 application?

### DISCUSSION THREAD

**Discussion by nyngwang** [TanStack/query#6862] [Q&A] [answered]
As title. Should I install TanStack/query since it's the latest version?

**Comment by Fullchee** [comment, marked as answer]
You should install the latest version (v5 as of early 2024) of the react flavour of @tanstack/query!

```
npm i @tanstack/react-query
```

https://tanstack.com/query/latest/docs/framework/react/installation


and optionally the dev tools too

https://tanstack.com/query/latest/docs/framework/react/devtools

```
npm i @tanstack/react-query-devtools
```

**Reply by nyngwang** [reply]
lol, I was fascinated by the landing page of TanStack and never tried to find the a-few-link-away docs you provided. Thank you!

**Reply by Fullchee** [reply]
Happy to help! 😃
